dream_layer_backend/img2img_workflow.py

```
@@ -0,0 +1,258 @@
+    batch_size = max(1, min(8, int(data.get('batch_size', 1))))
+    steps = max(1, min(150, int(data.get('steps', 20))))
+    cfg_scale = max(1.0, min(20.0, float(data.get('cfg_scale', 7.0))))
+    denoising_strength = max(0.0, min(1.0, float(data.get('denoising_strength', 0.75))))
+    input_image = data.get('input_image', '')
+    model_name = data.get('model_name', 'v1-6-pruned-emaonly-fp16.safetensors')
+    sampler_name = data.get('sampler_name', 'euler')
+    scheduler = data.get('scheduler', 'normal')
+    
+    # Advanced settings
+    vae_name = data.get('vae_name')
+    clip_skip = data.get('clip_skip', 1)
```

Ruff (0.11.9)

Local variable `clip_skip` is assigned to but never used

Remove assignment to unused variable `clip_skip`

(F841)